net2/IntelTool.js

```diff
@@ -162,11 +162,16 @@ class IntelTool {
     return rclient.delAsync(this.getURLIntelKey(url));
   }
 
-  updateHashMapping(hashCache, hash) {
+  updateHashMapping(hashCache, hash, ip) {
     if(Array.isArray(hash)) {
-      const origin = hash[0]
-      const hashedOrigin = hash[2]
-      hashCache[hashedOrigin] = origin
+      if (hash.length === 2) {
```

Comment:
no need to have length ifelse statements if updateHashMapping is called on _iplist instead of hds